Changes

```diff
@@ -11,6 +11,10 @@ _______________
 
 ### Standard library:
 
+- #12869: Add List.take, List.drop, List.take_while and List.drop_while
+  (Kate Deplaix, review by Nicolás Ojeda Bär, Craig Ferguson,
+   Gabriel Scherer and Oscar Butler-Aldridge)
```

Comment:
(It's a minor point but I would move Oscar to co-author.)

Reply:
It's appreciated. I don't mind either way. Whatever's easiest for you @kit-ty-kate :)

Reply:
done

Reply:
Not quite yet!

```
Checking c4bce87a16c60d8a128787929f0c977f317ba4d4: Changes
./Changes:15.81: [long-line] line is over 80 columns
```

Reply:
fixed.